src/CMakeLists.txt

```diff
@@ -4,19 +4,14 @@ find_package(Boost 1.56 REQUIRED)
 find_package(CURL       REQUIRED)
 find_package(MiniZip    REQUIRED)
 find_package(Threads    REQUIRED)
+find_package(unofficial-sqlite3 REQUIRED)
```

Comment:
Need to check whether the `CONFIG` keyboard is required to select the right debug or release version on Windows (as per the previous comment). There were issues with this in the past, but maybe vcpkg improved since then.

Reply:
Related to the previous question: I believe that all-in-vcpkg approach is better as we don't need to handle different versions of libraries possibly installed on user's machine. We can always link against vcpkg's sqlite otherwise.

Reply:
And about the release/debug differentiation: I believe it's no use building dependencies in Debug mode, as you generally assume they are bug-free.

Reply:
The problem was CMake used to choose vcpkg's debug SQlite DLL when making release builds (vcpkg installs both). Using `CMAKE_MAP_IMPORTED_CONFIG_*` and replacing `find_package(SQLite3 REQUIRED)` with `find_package(SQLite3 CONFIG REQUIRED)` was the solution for that (was before vcpkg renamed it `unofficial-sqlite3`). I don't know if that's still the case today.